src/main/resources/config.yml

```diff
@@ -1,4 +1,4 @@
-#zh_CN | en_US | ko_KR
+#zh_CN | en_US | ko_KR | vi_VIE
```

Comment:
Sorry, I don’t know Vietnamese well, but the abbreviation seems to be vi_VN

Reply:
You are right. Sorry I already forgot it :D

I will correct it